netmap/netmap_contract.go

```diff
@@ -263,7 +263,8 @@ func AddPeerIR(nodeInfo []byte) {
 
 	common.CheckAlphabetWitness(common.AlphabetAddress())
 
-	publicKey := nodeInfo[2:35] // V2 format: offset:2, len:33
+	keyOffset := 2
```

Comment:
You better define this as a constant, otherwise we'll have this in the resulting bytecode.

Reply:
Fixed.